caracal.js

```diff
@@ -49,7 +49,7 @@ app.use('/img/IIP/raw/', iipHandler);
 
 // loader, proxy
 app.use('/loader/', auth.loginHandler(auth.PUBKEY));
-app.use('/loader/', permissionHandler(['Admin']));
+app.use('/loader/', permissionHandler(['Admin', 'Editor']));
```

Comment:
I _think_ this should work as expected to limit deletes
```suggestion
app.use('/loader/', permissionHandler(['Admin', 'Editor']));
app.use('/loader/slide/delete', permissionHandler(['Admin']));
```

Reply:
Ah, yes. Thanks I missed it.